packages/mobile/src/features/videos/components/FiltersScreen/ShortcutToFilters.tsx

```diff
@@ -57,12 +49,13 @@ const SearchbarWrapper = styled.View({
   elevation: "3",
 });
 
-const InputText = styled(MyText)({
+const InputText = styled.TextInput({
```

Comment:
sprawdź prosze , bo wydaje mi się ze na masterze jest to juz poprawione

Reply:
mam tutaj kłopot ponieważ przy styled.(MyText) nie działa mi input i placholder...;/